src/components/components/FeaturedJobCard/FeaturedJobCard.scss

```diff
@@ -0,0 +1,84 @@
+.FeaturedJobCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 27px 24px;
+  gap: 1em;
+  border-radius: 1em;
+  width: 25em;
+  height: 20.5em;
+  background-color: var(--color-white);
+  border: 1px dashed red;
```

Comment:
I don't think we need a red border here, do we?

Reply:
sure, I forgot to remove))
I usually add it for testing purposes - to see all elements on browser